frontend/src/config/index.ts

```diff
@@ -87,6 +87,10 @@ const safeCountry =
     ? (COUNTRY.toLocaleLowerCase() as Country)
     : DEFAULT;
 
+// list countries that have a preprocessed-layer-dates.json file
+// to avoid a failed network call on each layer activation
+const countriesWithPreprocessedDates = ['cameroon', 'nigeria', 'rbd'];
```

Comment:
I like the idea, but is there a way to be smarter about that? And maybe create this list dynamically? Is theere a flag in prism.json we could use?

Reply:
yep, good call. I've added a few lines in the script that produces the preprocessed dates to also generate that list, so they should be in sync without extra action.